03-task-manager/starter/controllers/task.js

```diff
@@ -0,0 +1,27 @@
+const getAllTasks = (req, res) => {
+    res.send("get all items");
+};
+
+const createTask = (req, res) => {
```

Comment:
Around the 1:22:51 mark, the video covers the implementation of the create task
https://www.youtube.com/watch?v=rltfdjcXjmk&t=4587s

This needs to be implemented to confirm that the MongoDB database can be created by the code.

Reply:
I think it's now work properly